src/main/views/pending-subscriptions.njk

```diff
@@ -40,48 +40,52 @@
     </form>
```

Comment:
Needs some unit tests adding for this + the other file

Reply:
unit tests have been added for both pending and confirmation subscription pages.